src/utils/include/com/amazonaws/kinesis/video/utils/Include.h

```diff
@@ -1453,6 +1453,140 @@ PUBLIC_API STATUS freeFileLogger();
+    *     4       *   2400ms + jitter  *
+    *     5       *   4800ms + jitter  *
+    *     6       *   9600ms + jitter  *
+    *     7       *  10000ms + jitter  *
+    *     8       *  10000ms + jitter  *
+    *     9       *  10000ms + jitter  *
+    ************************************
+ jitter = random number between [0, 300)ms.
+************************************************************************/
+/**
+ * Factor for computing the exponential backoff wait time
+ */
+#define DEFAULT_KVS_RETRY_TIME_FACTOR_MILLISECONDS                  300
+
+/**
+ * Factor determining the curve of exponential wait time
+ */
+#define DEFAULT_KVS_EXPONENTIAL_FACTOR                              2
+
+/**
+ * Maximum exponential wait time. Once the exponential wait time
+ * curve reaches this value, it stays at this value. This is
+ * required to put a reasonable upper bound on wait time.
+ */
+#define DEFAULT_KVS_MAX_WAIT_TIME_MILLISECONDS                      15000
+
+/**
+ * Maximum time between two consecutive calls to exponentialBackoffBlockingWait
+ * after which the retry count will be reset to initial state.
+ */
+#define DEFAULT_KVS_MIN_TIME_TO_RESET_RETRY_STATE_MILLISECONDS      25000
+
+/**
+ * Factor to get a random jitter. Jitter values [0, 300).
+ */
+#define DEFAULT_KVS_JITTER_FACTOR_MILLISECONDS                      300
+
+typedef enum {
+    BACKOFF_NOT_STARTED     = (UINT16) 0x01,
+    BACKOFF_IN_PROGRESS     = (UINT16) 0x02,
+    BACKOFF_TERMINATED      = (UINT16) 0x03
+} ExponentialBackoffStatus;
+
+typedef struct __ExponentialBackoffConfig {
+    UINT32  maxRetryCount;
+    UINT64  maxWaitTime;
+    UINT32  retryFactorTime;
+    UINT64  minTimeToResetRetryState;
+    UINT32  jitterFactor;
+} ExponentialBackoffConfig;
+typedef ExponentialBackoffConfig* PExponentialBackoffConfig;
+
+typedef struct __ExponentialBackoffState {
+    PExponentialBackoffConfig pExponentialBackoffConfig;
+    ExponentialBackoffStatus status;
+    UINT32 currentRetryCount;
+    UINT64 lastRetryWaitTime;
+} ExponentialBackoffState;
+typedef ExponentialBackoffState* PExponentialBackoffState;
+
+/**************************************************************************************************
+API usage:
+
+ PExponentialBackoffState pExponentialBackoffState = NULL;
+ CHK_STATUS(createExponentialBackoffStateWithDefaultConfig(&pExponentialBackoffState));
+
+ while (...) {
+     CHK_STATUS(exponentialBackoffBlockingWait(pExponentialBackoffState));
+    // some business logic which includes service API call(s)
+ }
+
+ CHK_STATUS(exponentialBackoffStateFree(&pExponentialBackoffState));
+
+**************************************************************************************************/
+
+/**
+ * @brief Initializes exponential backoff state with default configuration
+ * This should be called once before calling exponentialBackoffBlockingWait.
+ *
+ * NOT THREAD SAFE.
+ */
+PUBLIC_API STATUS createExponentialBackoffStateWithDefaultConfig(PExponentialBackoffState*);
```

Comment:
To be consistent with the other create APIs like `timerQueueCreate` and `stackQueueCreate` and `hashTableCreate` etc. we should put `create` at the end.